src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -637,7 +637,8 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
           margin-top: 0;
         }
         ha-fab {
-          position: fixed;
+          position: sticky;
```

Comment:
We should not make it sticky, that means it will be under the content instead of at the bottom of your screen when there is little content. We should do it the same as we do on the other pages:
```css
        ha-fab[is-wide] {
          bottom: 24px;
          right: 24px;
        }
```

Reply:
24px high? Doesn't that go against the material design [spec](https://material.io/components/buttons-floating-action-button#specs)?

Done anyway to match the rest